styles/components/_layout.scss

```diff
@@ -90,6 +90,30 @@
   flex-direction: column;
 }
 
+.pureFlex {
+  display: flex;
```

Comment:
okay. I can do it like the way you say in 1. and I can also see a couple of reasons to do it the way in these changes. 1 may lead to a larger number of style rules to hold the combinations.  smaller rules have clear pupose and combine together in a flexible way . it's a trade-off.  also existing rules define properties unneeded to achieve the layouts in this case. I don't think it works to include those unnecessary properties. for example, absolute positioning and more rules when it can be done using flex is more brittle. finally, I totally agree with 2 and it's about the trade-off. for example, if some existing rules did not style optimally could they not be improved? and if having smaller rules that fit together in flexible combinations made it easier to add new layout and styles that maintained consistency is that not something that works?

when there's time I'll try to present these ideas more closely in some finished code and we can see if it's a contribution that works.